compose.go

```diff
@@ -113,11 +113,12 @@ func (dc *LocalDockerCompose) applyStrategyToRunningContainer() error {
 	}
 
 	for k := range dc.WaitStrategyMap {
-		// TODO: if the compose file includes a container_name, this won't work,
-		// as we cannot delegate to the names generated by Docker Compose
 		containerName := dc.Identifier + "_" + k.service
-		// Docker compose appends "_1" to every started service by default. Trimming to match running docker container
-		f := filters.NewArgs(filters.Arg("name", containerName))
+		composeV2ContainerName := strings.Replace(containerName, "_", "-", -1)
```

Comment:
```suggestion
		composeV2ContainerName := strings.ReplaceAll(containerName, "_", "-")
```

Reply:
If this line is blocker for merge, I can create another pull request with this change. Is it blocker ?

Reply:
Not a blocker at all, as I could directly commit it to this PR, but wanted to give @artamonovkirill feedback and time to see it before committing it